tests/test_xlayer_cs.py

```diff
@@ -1,10 +1,12 @@
 # SPDX-FileCopyrightText: Copyright DB Netz AG and the capellambse contributors
 # SPDX-License-Identifier: Apache-2.0
 
+import pytest
+
 from capellambse import MelodyModel
 
 
-def test_physical_path_has_ordered_list_of_involved_items(model: MelodyModel):
+def test_PhysicalPath_has_ordered_list_of_involved_items(model: MelodyModel):
```

Comment:
Thank you for also making all names of the tests consistent. This keeps my sanity up high.